draft-ietf-grow-nrtm-v4.xml

```diff
@@ -142,7 +142,9 @@
        </list>
      </t>
      <t>
-       All files MUST be in the JavaScript Object Notation (JSON) format <xref target="RFC4627" />.
+       The Update Notification file MUST be in the JavaScript Object Notation (JSON) format <xref target="RFC4627" />.
```

Comment:
I suggest replacing reference to RFC4627 (now obsolete) with the latest version RFC8259

Reply:
👍🏻